social_auth/backends/contrib/odnoklassniki.py

```diff
@@ -189,7 +189,7 @@ def get_response(self):
                   'apiconnection',
                   )
         response = {}
-        for fieldname in self.fields.keys():
+        for fieldname in list(self.fields.keys()):
```

Comment:
`.keys()` on a dict already return a list with the name of the props in the dict